common/entryCreditBlock/commitchain.go

```diff
@@ -37,39 +37,39 @@ var _ interfaces.ShortInterpretable = (*CommitChain)(nil)
 var _ interfaces.IECBlockEntry = (*CommitChain)(nil)
 var _ interfaces.ISignable = (*CommitChain)(nil)
 
-func (e *CommitChain) Init() {
-	if e.MilliTime == nil {
-		e.MilliTime = new(primitives.ByteSlice6)
+func (c *CommitChain) Init() {
```

Comment:
(IntelliJ warned that the var name was inconsistent and asked if it could fix it.)